Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM node:16-alpine AS node
```

Comment:
Is there a way we could use a later version of node?

Reply:
this is the one recommended by subsquid, is there any benefits on trying out a newer one?